src/dispatch.js

```diff
@@ -11,6 +11,42 @@ import {
   ifElse,
 } from 'ramda';
 
+/**
+ * Can be used as a way to compose multiple invokers together to form polymorphic functions,
+ * or functions that exhibit different behaviors based on their argument(s).
+ * Consumes dispatching functions and keep trying to invoke each in turn, until a non-nil value is returned.
+ *
+ * Accepts a list of dispatching functions and returns a new function.
+ * When invoked, this new function is applied to some arguments,
+ * each dispatching function is applied to those same arguments until one of the
+ * dispatching functions returns a non-nil value.
+ *
+ * @func dispatch
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.6.0|v2.6.0}
+ * @category Function
+ * @sig [((a, b, ...) -> x1), ((a, b, ...) -> x2), ...] -> x1 | x2 | ...
+ * @param {!Array} functions A list of functions
+ * @return {*|undefined} Returns the first not-nil value or undefined if empty list provided
```

Comment:
Add note about what happens if no function returns a non-nil value.

Reply:
Maybe this one is simple to read ?

`Returns the first not-nil value, or undefined if either an empty list is provided or none of the dispatching functions returns a non-nil value.`

Reply:
LGTM